ipaserver/install/cainstance.py

```diff
@@ -271,6 +273,30 @@ def create_ca_user():
     )
 
 
+def ra_cert_retrieval_wrapper(ra_cert_retrieval):
+    """
+    This is a decorator which helps to backup RA agent files that could be
+    left in the system from previous installations.
+    It sets the right permissions to the newly received files.
+    """
+    def wrapper():
+        # first back up the RA agent files
+        for fname in (paths.RA_AGENT_PEM, paths.RA_AGENT_KEY):
+            if os.path.exists(fname):
+                os.rename(fname, fname + '.bcp')
```

Comment:
IMO we should not backup the files, they are in `/var/lib/ipa` and noone except IPA itself has any bussiness touching them.